experiments/compositions/composition/config/manager/kustomization.yaml

```diff
@@ -18,5 +18,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: gcr.io/wfender-gke-dev/composition
-  newTag: 0.0.18
+  newName: gcr.io/allotrope-barni/composition
```

Comment:
We need to get rid of both 'wfender-gke-dev' and 'barni'. Probably need to run a templating engine or sed script to set the appropriate project here.

Reply:
I have a way of doing it temporarily in makefile.

Reply:
done